### PR TITLE
MESH-795: AccountKey removal

### PR DESCRIPTION
**Pull request by vkomenda**

### CODE REVIEW

File: src/chain_specs/aldebaran.json [hunk 1]
```diff
@@ -386,19 +386,19 @@
         "creator": "5GGesvoesHcWwuvTJhLkfoxd39DwaS2K49PENQL8f6Ax91D2",
         "signers": [
           {
-            "AccountKey": "0x9870910d65ccb11af1c85d8ed1a17bc2229bbc08edf44c0dc7865e0edbef8b35"
+            "Account": "0x9870910d65ccb11af1c85d8ed1a17bc2229bbc08edf44c0dc7865e0edbef8b35"
```
Comment (maxsam4) [inline src/chain_specs/aldebaran.json:389]: Past chain specs should not be changed. This file is present only for "preservation".

Comment (vkomenda) [reply src/chain_specs/aldebaran.json:389]: OK, I'll roll it back.

File: pallets/identity/src/lib.rs [hunk 2]
```diff
@@ -1367,68 +1393,68 @@ impl<T: Trait> Module<T> {
...
     }
 
     /// Update link data (if it exists) from a key or identity.
     /// NB: Please do all the required checks before calling this function.
-    pub fn update_link(target: Signatory, link_id: u64, link_data: LinkData) {
-        if <Links<T>>::contains_key(target, link_id) {
-            <Links<T>>::mutate(target, link_id, |link| link.link_data = link_data);
+    pub fn update_link(target: Signatory<T::AccountId>, link_id: u64, link_data: LinkData) {
+        if <Links<T>>::contains_key(&target, link_id) {
+            <Links<T>>::mutate(target.clone(), link_id, |link| link.link_data = link_data);
```
Comment (fmiguelgarcia) [inline pallets/identity/src/lib.rs:1411]: Could we use `target` as ref here?

File: pallets/runtime/common/src/contracts_wrapper.rs [hunk 3]
```diff
@@ -28,17 +28,15 @@
...
 use frame_system::ensure_signed;
 use pallet_contracts::{CodeHash, Gas, Schedule};
+use pallet_identity as identity;
+use polymesh_common_utilities::{identity::Trait as IdentityTrait, Context};
+use polymesh_primitives::{IdentityId, Signatory};
 use sp_runtime::traits::StaticLookup;
-use sp_std::{convert::TryFrom, prelude::*};
+use sp_std::prelude::*;
 
 // pub type CodeHash<T> = <T as frame_system::Trait>::Hash;
```
Comment (fmiguelgarcia) [inline pallets/runtime/common/src/contracts_wrapper.rs:41]: To be deleted ?

**Pull request merged**